ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -31,7 +31,9 @@
 #include <fstream>
 #include <list>
 #include <errno.h>
+#include <list>
```

Comment:
The list is already included on line 32, and since the lock on active file systems worked without including mutex header, I wonder if we need the inclusion on line 36.

Reply:
Yup, it compiles fine without the <mutex> include.